src/Cdev/Command/InstallDependenciesCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Cdev\Local\Cdev\Command;
+
+use Symfony\Component\Process\Process;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class InstallDependenciesCommand extends Command {
+    protected function configure()
+    {
+        $this->setName('local:setup-dependencies');
```

Comment:
all other environment commands are prefixed with `env:` so should use that here, that'll mean it's grouped up properly when the command list is viewed